### DISCUSSION TITLE
01. What is open source?

### DISCUSSION THREAD

**Discussion by MichaelRimler** [phuse-org/OSTCDA#1] [General]
What do we mean by "**open source**"? What is open source?

**Comment by mikecarniello** [comment]
It depends! This question reflects the tension in the community. It's free - but is it free like "free beer" or free like "free speech?"

More tension: in our pharma world, are we building cathedrals or a bazaars? (see the classic work: http://www.catb.org/~esr/writings/cathedral-bazaar/cathedral-bazaar/) (spoiler: SAS has been our cathedral!)

"Open source" thinking and work has not been planned nor designed; it's evolved from hacker types doing "cool" stuff into being used for life and death decisions (our pharma world).

**Comment by KatjaGlassConsulting** [comment]
Open Source Definition by the "Open Source Initiative": https://opensource.org/osd/

Core points:
- free distribution
- available source code
- derived works possible

**Comment by KatjaGlassConsulting** [comment]
Differentiation required between "using" open source (consumer) and "creating/supporting" open source (contributor) might be done.

A consumer of open source is for example a company using R and various open source packages, whereas a contributor is also publishing open source or collaborating in open source project (contribute, feedback, discussions).

**Comment by MichaelRimler** [comment]
According to [Merriam-Webster](https://www.merriam-webster.com/dictionary/open-source), open source is “having the source code freely available for possible modification and redistribution”. [Wikipedia](https://en.wikipedia.org/wiki/Open-source_software) defines open-source software (OSS) as a computer software as computer software that is released under a license in which the copyright holder grants users the rights to use, study, change and distribute the software and its source code to anyone and for any purpose.

**Comment by joseph-rickert** [comment]
It is "free" like a "free puppy". It comes with a great deal of responsibility.
